### DISCUSSION TITLE
Azure segment - show account

### DISCUSSION THREAD

**Discussion by blauwkeut** [JanDeDobbeleer/oh-my-posh#721] [Ideas]
Was wondering if it's possible to also be able to show the signed in account in the Azure segment, besides ID and Name? Account is a bit more meaningfull (in my situation) than the name...

![2021-05-19 15_48_25-powershell -- C__temp](https://user-images.githubusercontent.com/33721771/118824586-440c0b80-b8ba-11eb-8cbf-56599d2fd241.png)

Cheers,
Dennis

**Comment by JanDeDobbeleer** [comment]
I'll have a look!

**Reply by JanDeDobbeleer** [reply]
@blauwkeut can you confirm, when using the `az` CLI, this is the same value as what you'd see as `user.name`?

**Reply by JanDeDobbeleer** [reply]
You now have the [ability](https://ohmyposh.dev/docs/az#properties) to also add the subscription account.

**Reply by blauwkeut** [reply]
Thanks for looking into it... Have updated to latest, but for some reason (not sure if it's my environment or OMP) it returns a GUID rather than a 'human readable' account name...

FWIW - I now also noticed both display_id and display_account actually return the same GUID string:
![2021-05-25 10_29_53-powershell -- ~](https://user-images.githubusercontent.com/33721771/119465768-45b75280-bd44-11eb-82ae-09c8a3840199.png)

It seems the GUID is the one from my subscription:
![2021-05-25 10_36_34-powershell -- ~](https://user-images.githubusercontent.com/33721771/119466683-2a007c00-bd45-11eb-9941-c8ea4643f810.png)

**Reply by JanDeDobbeleer** [reply]
@blauwkeut fix incoming. Made a mistake (and didn't add the proper tests) 🤦🏻
